lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -24,10 +24,10 @@ import {
 import { Configuration, buildAppConfiguration } from '../config/Configuration';
 import { CryptoProvider } from '../crypto/CryptoProvider';
 import { Storage } from '../cache/Storage';
-import { version } from '../../package.json';
 import { Constants as NodeConstants } from './../utils/Constants';
 import { TokenCache } from '../cache/TokenCache';
 import { ClientAssertion } from "../client/ClientAssertion";
+const pjson = require("../../package.json");
```

Comment:
I thought we are avoiding `require` except in tests.

Reply:
+1

Reply:
Hmm then we may need to think of something because we can't import package.json from outside the root

Reply:
I think it's ok for this scenario. Did some testing and there isn't any way this will work currently because `tsdx` has issues with `.` as the rootDir. This is one reason I don't think we should have used `tsdx`.

Reply:
So even adding a `@ts-ignore` in order to import, silently changes the `rootDir` back to `./` in the background. We'll move forward with the `require` for now and revisit later